storage-proofs-porep/src/stacked/vanilla/proof.rs

```diff
@@ -592,6 +688,7 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
 
         #[cfg(feature = "multicore-sdr")]
         {
+            use storage_proofs_core::settings::SETTINGS;
```

Comment:
I guess those are added due to Clippy warnings. What about importing it at the top again as:

```rust
#[cfg(any(feature = "cuda", feature = "multicore-sdr", feature = "opencl"))]
use storage_proofs_core::settings::SETTINGS;
```